demo/screens/bottomnavigation.py

```diff
@@ -18,14 +17,15 @@
 
         AKBottomNavigation:
             items: root.bottomnavigation_items
+
 '''
 )
 
 class BottomNavigation(Screen):
 
     bottomnavigation_items=[
-        {'icon':'android' , 'text': 'android' , 'on_release': lambda x: toast('android')},
```

Comment:
What has this to do with the main purpose?

Reply:
sorry that was a my mistake, I never intended to post that too